package/src/api.ts

```diff
@@ -49,6 +94,10 @@ export interface GetVersionsOptions<
    * @default undefined
    */
   after?: string
+  /**
+   * Retry options for the built-in retry mechanism
+   */
+  retryOpts?: RetryOptions
```

Comment:
```suggestion
  retry?: RetryOptions | number | false
```

I prefer either `retry` or `retryOptions`